examples/inbound-webhook-parsing/app.py

```diff
@@ -0,0 +1,54 @@
+from flask import Flask, request, Response
+import os
+
+HOST = '0.0.0.0'
+PORT = 5000
+
+app = Flask(__name__)
+
+
+@app.route('/')
+def hello():
+    print("testing")
+    return Response("Up and running", status=200)
+
+
+@app.route('/parse_inbound', methods=['POST'])
+def inbound_parser():
+    data = request.form
+    output = ""
+
+    output += "------------------ INCOMING WEBHOOK DATA ------------------"
+    output += "\n\n"
+    output += "[HEADERS]\n"
+    output += data['headers']
+    output += "\n\n"
+    output += "[DKIM]\n"
+    output += data['dkim']
+    output += "\n\n"
+    output += "[FROM]\n"
+    output += data['from']
+    output += "\n\n"
+    output += "[TO]\n"
+    output += data['to']
+    output += "\n\n"
+    output += "[SUBJECT]\n"
+    output += data['subject']
+    output += "\n\n"
+    output += "[BODY]\n"
+    output += data['text']
+    output += "\n\n"
+    output += "[ATTACHMENTS]\n"
+    output += data['attachments']
+    output += "\n\n"
+    output += "[CHARSETS]\n"
+    output += data['charsets']
+    output += "\n\n"
+    output += "------------------ END WEBHOOK DATA ------------------"
```

Comment:
It seems slightly verbose to create a multi-line string this way? I know the README file above mentions Python 3.6+ which could give the option of using f-strings but even just `str.format` might be able to clean it up a bit?

```python
In[2]: import textwrap
  ...: 
  ...: data = {
  ...:     'headers': 'headers',
  ...:     'dkim': 'dkim',
  ...:     'from': 'from',
  ...:     'to': 'to',
  ...:     'subject': 'subject',
  ...:     'text': 'text',
  ...:     'attachments': 'attachments',
  ...:     'charsets': 'charsets'
  ...: }
  ...: 
  ...: 
  ...: def inbound_parser():
  ...:     output = ""
  ...:     output += "------------------ INCOMING WEBHOOK DATA ------------------"
  ...:     output += "\n\n"
  ...:     output += "[HEADERS]\n"
  ...:     output += data['headers']
  ...:     output += "\n\n"
  ...:     output += "[DKIM]\n"
  ...:     output += data['dkim']
  ...:     output += "\n\n"
  ...:     output += "[FROM]\n"
  ...:     output += data['from']
  ...:     output += "\n\n"
  ...:     output += "[TO]\n"
  ...:     output += data['to']
  ...:     output += "\n\n"
  ...:     output += "[SUBJECT]\n"
  ...:     output += data['subject']
  ...:     output += "\n\n"
  ...:     output += "[BODY]\n"
  ...:     output += data['text']
  ...:     output += "\n\n"
  ...:     output += "[ATTACHMENTS]\n"
  ...:     output += data['attachments']
  ...:     output += "\n\n"
  ...:     output += "[CHARSETS]\n"
  ...:     output += data['charsets']
  ...:     output += "\n\n"
  ...:     output += "------------------ END WEBHOOK DATA ------------------"
  ...:     return output
  ...: 
  ...: 
  ...: def inbound_parser_fstring():
  ...:     output = textwrap.dedent(f"""\
  ...:         ------------------ INCOMING WEBHOOK DATA ------------------
  ...: 
  ...:         [HEADERS]
  ...:         {data['headers']}
  ...: 
  ...:         [DKIM]
  ...:         {data['dkim']}
  ...: 
  ...:         [FROM]
  ...:         {data['from']}
  ...: 
  ...:         [TO]
  ...:         {data['to']}
  ...: 
  ...:         [SUBJECT]
  ...:         {data['subject']}
  ...: 
  ...:         [BODY]
  ...:         {data['text']}
  ...: 
  ...:         [ATTACHMENTS]
  ...:         {data['attachments']}
  ...: 
  ...:         [CHARSETS]
  ...:         {data['charsets']}
  ...: 
  ...:         ------------------ END WEBHOOK DATA ------------------""")
  ...:     return output
  ...: 
  ...: 
  ...: def inbound_parser_format():
  ...:     output = textwrap.dedent("""\
  ...:         ------------------ INCOMING WEBHOOK DATA ------------------
  ...: 
  ...:         [HEADERS]
  ...:         {headers}
  ...: 
  ...:         [DKIM]
  ...:         {dkim}
  ...: 
  ...:         [FROM]
  ...:         {from}
  ...: 
  ...:         [TO]
  ...:         {to}
  ...: 
  ...:         [SUBJECT]
  ...:         {subject}
  ...: 
  ...:         [BODY]
  ...:         {text}
  ...: 
  ...:         [ATTACHMENTS]
  ...:         {attachments}
  ...: 
  ...:         [CHARSETS]
  ...:         {charsets}
  ...: 
  ...:         ------------------ END WEBHOOK DATA ------------------""".format(**data))
  ...:     return output
  ...: 
In[3]: inbound_parser() == inbound_parser_fstring() == inbound_parser_format()
Out[3]: True
```

Reply:
Yeah sure I can check it out

Reply:
Decided to go the str.format route for better compatibility. Let me know if there are any other issues.